src/commands/init/index.ts

```diff
@@ -110,6 +110,7 @@ Velocitas project found!
                 }
             }
         } else {
```

Comment:
Not a big fan of nesting like this, we should keep it short and precise. I would flip the if-condition in line 87 to `if !ProjectConfig.isAvailable())` and then create the project config. We wouldn't even need to exit then and just continue with project initialization.